### PR TITLE
13193-fss-uI-change-in-error-message-with-response-429 AB#13210 AB#13226

### PR DESCRIPTION
**Pull request by Rajesh14881**
Change in error message with 429 response

### CODE REVIEW

File: src/app/features/fss-search/fss-search.component.ts [hunk 1]
```diff
@@ -204,9 +204,12 @@ export class FssSearchComponent implements OnInit {
       errmsg = err.error.message;
     }
     if(this.activeSearchType == this.SearchTypeEnum.SimplifiedSearch)
-      {
-        this.showMessage("error", "There has been an error", "please contact customer services");
-      }
+      if(err.error.statusCode == this.TooManyRequest){
```
Comment (crdandoukho) [inline src/app/features/fss-search/fss-search.component.ts:207]: Does the changed error message apply to only simplified search? It should probably also apply to advanced search if not already.

Could you please also tidy up the `if` statement/blocks

**Review by crdandoukho — CHANGES_REQUESTED** [review]
A few comments

File: src/app/features/fss-search/fss-search.component.ts (hunk 1 above)
Comment (bhalcha13813) [reply src/app/features/fss-search/fss-search.component.ts:207]: In simplified search common message is shown to contact administrator and that is why we explicitly checked for 429.  
whereas In advanced search whatever response we get from API that is shown.

Comment (crdandoukho) [reply src/app/features/fss-search/fss-search.component.ts:207]: Yes please handle it the same way for Advanced Search too

**Review by bhalcha13813 — APPROVED** [review]
LGTM

**Review by crdandoukho — APPROVED** [review]
Looks good

**Pull request merged**